### PR TITLE
Tag Interact.jl v0.4.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaGizmos/Interact.jl](https://github.com/JuliaGizmos/Interact.jl)
Release: [v0.4.0](https://github.com/JuliaGizmos/Interact.jl/releases/tag/v0.4.0)
Diff: [vs v0.3.5](https://github.com/JuliaGizmos/Interact.jl/compare/522151c7405ef3a52ab64ccbb228411bd240d240...a35440f45b429ed7a9b19c98b55e174380a75524)
`requires` vs v0.3.5: 
```diff
--- 0.3.5/requires
+++ 0.4.0/requires
@@ -2,4 +2,4 @@
 JSON
 Compat 0.8.0
 Reactive 0.3
-DataStructures 0.2.10
+DataStructures
```
cc: @shashi

### CODE REVIEW

**Comment by shashi** [comment]
merge after #7385

**Comment by tkelman** [comment]
why was the datastructures minimum version removed?

**Comment by tkelman** [comment]
looks like you didn't re enable travis after you moved the repo

File: Interact/versions/0.4.0/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.4
+JSON
```
Comment (tkelman) [inline Interact/versions/0.4.0/requires]: JSON.lower needs a minimum version

**Comment by tkelman** [comment]
if this requires a minimum version of IJulia to work, then the check for IJulia installation here should account for that. It's possible to update individual packages independently now.

**Comment by shashi** [comment]
I've addressed the comments thanks @tkelman 

The tests pass but the test script returns a non-zero exit code (`1`) for some reason. This should be safe to merge now, we can deal with the test problem after that.

**Comment by shashi** [comment]
tests also fixed, build passes.

**Pull request merged**